aiohttp/__init__.py

```diff
@@ -93,7 +93,6 @@
     EMPTY_PAYLOAD as EMPTY_PAYLOAD,
     DataQueue as DataQueue,
     EofStream as EofStream,
-    FlowControlDataQueue as FlowControlDataQueue,
```

Comment:
I couldn't find anything on GitHub code searching using this externally, and I don't think it would make too much sense to do so since it was only used for the WebSocket anyways.